examples/crna-kitchen-sink/package.json

```diff
@@ -3,15 +3,16 @@
   "version": "0.1.0",
   "private": true,
   "devDependencies": {
-    "@storybook/addon-actions": "file:../../addons/actions",
-    "@storybook/addon-links": "file:../../addons/links",
-    "@storybook/addon-options": "file:../../addons/options",
-    "@storybook/addon-storyshots": "file:../../addons/storyshots",
-    "@storybook/addons": "file:../../lib/addons",
-    "@storybook/channels": "file:../../lib/channels",
-    "@storybook/channel-postmessage": "file:../../lib/channel-postmessage",
-    "@storybook/react-native": "file:../../app/react-native",
-    "@storybook/ui": "file:../../lib/ui",
+    "@storybook/addon-actions": "file:../../packs/storybook-addon-actions.tgz",
```

Comment:
@ndelangen thanks for pinging me on this.

My memory is a bit hazy at the moment, but I recall that this config + `build-packs` was something I added to at least one of the example projects to avoid symlinks due to the various problems with them. This setup avoids symlinks with npm@5+, and reproduces the old `file:...` behaviour on npm@4 (copying package contents over).

However, if we haven't needed it before now on this particular example app, I wonder why we need it now? Can you explain?